wordpress.org/public_html/wp-content/plugins/wporg-gp-customizations/inc/class-plugin.php

```diff
@@ -810,4 +828,99 @@ public function get_custom_reasons( $default_reasons, $locale ) {
 		$reasons = isset( $locale_reasons[ $locale ] ) ? $locale_reasons[ $locale ] : array();
 		return array_merge( $default_reasons, $reasons );
 	}
+
+	/**
+	 * Filter the permission to CRUD events for the user.
+	 *
+	 * wp-org-translation-events plugin.
+	 *
+	 * @param bool $can_crud_event Whether the user can CRUD events.
+	 *
+	 * @return bool Whether the user can CRUD events.
+	 */
+	public function gp_translation_events_can_crud_event( bool $can_crud_event ): bool {
+		$user = wp_get_current_user();
+
+		if ( GP::$permission->user_can( $user, 'admin' ) ) {
+			return true;
+		}
+		if ( current_user_can( 'manage_options' ) ) {
+			return true;
+		}
+		if ( self::is_user_an_wporg_gte( $user ) ) {
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * Indicates if the given user is a GTE at translate.wordpress.org.
+	 *
+	 * Caches the GTE email addresses for 12 hours.
+	 *
+	 * @param WP_User $user A user object.
+	 *
+	 * @return bool Whether the user is GTE for any of the languages to which the comments in the post belong.
+	 */
+	public static function is_user_an_wporg_gte( WP_User $user ): bool {
+		$locales             = GP_Locales::locales();
+		$gte_email_addresses = wp_cache_get( self::GTE_EMAIL_ADDRESSES, self::CACHE_GROUP );
+
+		if ( false === $gte_email_addresses ) {
+			foreach ( $locales as $locale ) {
+				$gte_email_addresses = self::get_gte_email_addresses( $locale->slug );
+			}
+
+			wp_cache_set( self::GTE_EMAIL_ADDRESSES, $gte_email_addresses, self::CACHE_GROUP, 12 * HOUR_IN_SECONDS );
+		}
+
+		if ( empty( array_intersect( array( $user->user_email ), $gte_email_addresses ) ) ) {
```

Comment:
I think using `in_array()` would be more explicit:

```suggestion
		if ( ! in_array( $user->user_email, $gte_email_addresses ) ) {
```

Reply:
Done!